configs/.golangci.yml

```diff
@@ -0,0 +1,2 @@
+issues:
```

Comment:
what this file will do?

Reply:
`.golangci.yml `is a configuration file used by `golangci-lint`

It can be used to mold the linter as per our repo needs